### PR TITLE
Fixing README.md.

### PR DESCRIPTION
**Pull request by nhosoi**
- Turning each example which was a snippet of playbook to a complete one.
- Fixing unmatched and missing highlighting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,25 +96,25 @@ logging_flows:
   - `state`: State of the configuration file. `present` or `absent`. Default to `present`.
 
 - `basics` type - `basics` input supports reading logs from systemd journal or systemd unix socket.<br>
-  available keys
+  **available keys**
```
Comment (pcahyna) [inline README.md:99]: can you please change **keys** to **options** ?

**Comment by pcahyna** [comment]
Another suggested fix: "Message Queue` (kafka, amqp) - Not yet implemented" If it is not implemented, why to mention it? Smame with Viaq. Moreover, the link to Viaq is dead.

**Comment by nhosoi** [comment]
Thanks, @pcahyna !
https://github.com/linux-system-roles/logging/pull/137/commits/7a81aab815ed2eabd408f36bcde056982f23a731

**Comment by nhosoi** [comment]
Thank you, @richm. Squashed.

**Pull request merged**